### ISSUE TITLE
add 'conference functionality' to teams app

### ISSUE THREAD

**Issue by FloorD** [rails-girls-summer-of-code/rgsoc-teams#114]
As the conference raffle-out did not prove to work last year around - the teams didn’t understand they had to tweet the conferences with the hashtag we provided, causing a lot of work checking all Twitter accounts - we thought of a different way. In the teams app participants can add a list of the conferences they would like to visit (min. 3 max. 10) - maybe drop-down select? All conferences should be linked to their Twitter accounts / hashtag so we would be able to create a default tweet text out of the conferences they’ve selected:

We would love to go to @conference, @conference and @conference #rgsocconf

**The conferences:**
Burlington Ruby Conference - August 1-3, Burlington, VT
@btvrubyconf
4 tickets

Steel City Ruby - August 15-16, Pittsburgh, PA
@steelcityruby
2 tickets

Madison Ruby Conference - August 22-23, Madison, WI
@madisonruby
2 tickets

Rulu - June 19-20, Lyon (France)
@rulu
2 tickets

eurucamp - August 1-3 Berlin
@eurucamp
2 tickets

Frozen Rails - September 11-12, Helsinki (Finland)
@frozenrails
2 tickets

Barcelona Ruby Conf - September 12-14, Barcelona (Spain)
@baruco
2 tickets

The Strange Loop - September 17-19, St. Louis (US)
@strangeloop_stl
2 tickets

Ruby DCamp -  October 10-12, US
@ruby_dcamp
2 tickets

Rails Israel - November 4-5, Tel Aviv (Israel)
#railsisrael
2 tickets

Arrrrcamp - October 2-3 Ghent (Belgium)
@arrrrcamp
2 tickets

PolyConf - October 30-31 Poznan (Poland)
@polyconfhq
2 tickets

GoTo Conference - November 6-7 Berlin
@gotober
2 tickets

It might make sense to look at the raffle machine’s code when creating this functionality: https://github.com/rails-girls-summer-of-code/rgsoc-teams/blob/master/lib/confs.rb#L7-L30

Please keep me updated on this so I can prepare a blog post announcing the raffle-out. :bowtie:

**Comment by alicetragedy** [comment]
I also think this could be useful :+1:

**Comment by ramonh** [comment]
I'm on it!

**Comment by alicetragedy** [comment]
@ramonh that's awesome!!!! :confetti_ball:

**Comment by ramonh** [comment]
Finally! I got some time to start taking a look at this. Right off the bat, I noticed that there's already some `conference` pages in the teams app, listing these. There are currently no conferences inserted, as shown below:

![screen shot 2015-06-10 at 17 16 18](https://cloud.githubusercontent.com/assets/656318/8086205/72f6de1e-0f94-11e5-8e9d-b242a31c3b11.png)

If I've understood the issue correctly, participating students should be allowed to add conferences they'd like to attend. Required fields would include the `twitter` and `hashtag`. What I'll also do is add a 'tweet' button that will generate the discussed pre-written tweet.

Is this what you had in mind @FloorD?
